packages/harmony/src/components/button/PlainButton/PlainButton.mdx

```diff
@@ -1,22 +1,100 @@
-import { Meta, Title, Description, Primary, Stories } from '@storybook/blocks'
+import { Meta, Title, Description, Controls, Canvas, Primary, Stories, Unstyled } from '@storybook/blocks'
 
-import { Heading, BetaWarning } from 'storybook/components'
+import { IconCaretDown, IconCloudUpload } from 'icons'
+import { ComponentRules, Heading, BetaWarning } from 'storybook/components'
 
+import { Box } from '../../layout'
+import { PlainButtonSize } from '../types'
+import { PlainButton } from './PlainButton'
 import * as PlainButtonStories from './PlainButton.stories'
 
 <Meta of={PlainButtonStories} />
 
-<Title>Plain Button</Title>
+<Title />
 
 <BetaWarning />
 
 - [Overview](#overview)
 - [Props](#props)
+- [Variants](#variants)
+- [Configuration options](#configuration-options)
+- [Use cases and examples](#use-cases-and-examples)
+- [Do's and dont's](#dos-and-donts)
+- [Related Components](#related-components)
```

Comment:
looks like there aren't any. Should add the section or remove the link

Reply:
Added 👍 . Good eye though